.gitignore

```diff
@@ -4,4 +4,8 @@ src/*.d
 src/*.o
 src/*.map
 *.bin
-*.elf
\ No newline at end of file
+*.elf
+/.cproject
+/.project
+/.vscode/
+/dirtyjtag Default.launch
```

Comment:
IDE-specific files shouldn't be added to DirtyJTAG's .gitignore. I recommend you to setup a global gitignore file for your system: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c ;-)